tests/integration/test_projectconfigs.py

```diff
@@ -419,3 +419,16 @@ def test_cached_project_config(mini_sentry, relay):
 
     assert data["configs"][public_key]["projectId"] == project_key
     assert not data["configs"][public_key]["disabled"]
+
+    # Wait till grace period expires as well and we should be dropping events now.
+    time.sleep(5)
```

Comment:
Could we shorten this `sleep` by settings `project_expiry` to 1 and `project_grace_period` to 2? Or would that make the test flaky?

Reply:
Yeah, that's why I gave it longer time here, since it depends on how fast you computer is, it can make this test flaky, which I would like to avoid. 
Once we port those integration tests to Rust we could make this work and look much more elegant.